lib/message/processMIME.js

```diff
@@ -154,5 +154,5 @@ const parse = async (
 export default async function processMIME(options, data) {
     const { subdata, verified, signatures } = await verifySignature(options, data);
 
-    return parse(options, subdata, verified, signatures);
+    return parse(options, subdata, verified, signatures || []);
```

Comment:
Shouldn't `verifySignature` always return a signatures array? Well.. except for
```js
    if (!rawboundary) {
        return data;
    }
```
but I think that should be 
```js
        return { subdata: data, verified: 0, signatures: [] };
```
Other than that, LGTM :+1:

Reply:
That sounds like the logic thing, but I ignored if
```
if (!rawboundary) {
    return data;
}
```
would have a deeper meaning :man_shrugging: . But it looks more like a typo, so let's do that!